chapters/annotations.tex

```diff
@@ -1031,9 +1199,32 @@ \subsubsection{Edit input}\doublelabel{edit-input}
 end OnMouseDownEditString;
 \end{lstlisting}
 \section{Annotations for the Graphical User Interface}\doublelabel{annotations-for-the-graphical-user-interface}
+This section describes the annotations that are used to define properties of the
```

Comment:
Why does it look like this unrelated annotation was cut and pasted instead of just text inserted before it?
Was there some change in the text (I didn't read that closely)? Was it moved?

Reply:
I think the changes are just minor, including adding a paragraph break before the `documentation-class-annotation` code block.

Reply:
OK, it is actually more than that.  Both the `documentation-class-annotation` and `preferred-view-annotation` are now described with the other _Annotations for the Graphical User Interface_.

Reply:
With the motivation that they control how the UI should handle the class and doesn't have anything to do with the actual documentation.

Reply:
@HansOlsson, I consider this resolved now. Feel free to reopen the discussion if you don't agree.